python/lsst/qserv/tests/qservDbLoader.py

```diff
@@ -78,8 +78,8 @@ def _callLoader(self, table):
         loaderCmd += ['--css-remove']
 
         if self.multi_node:
-            # Hard coded worker list
-            loaderCmd += ['--worker', 'worker1']
+            for node in self.nodeNames:
```

Comment:
Actually, I think it's more idiomatic (and portable to Python 3) to just have `for node in self.nWmgrs`.  See http://python-future.org/compatible_idioms.html#iterating-through-dict-keys-values-items